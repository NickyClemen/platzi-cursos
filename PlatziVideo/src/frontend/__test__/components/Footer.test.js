import React from 'react';

import { mount } from 'enzyme';

import { create } from 'react-test-renderer';

import Footer from '../../components/Footer';
// Una suite es una serie de pruebas.

describe('<Footer />', () => {
    const footer = mount(<Footer />);

    test('render footer component', () => {
        expect(footer.length).toEqual(1);
    });

    test('footer have three anchors', () => {
        expect(footer.find('a')).toHaveLength(3);
    });

    test('footer snapshot', () => {
        const footer = create(<Footer />);
        expect(footer.toJSON()).toMatchSnapshot();
    });
}); /* Palabra reservada.
Recibe una descripción de lo que se va a probar, y una función anónima. */
