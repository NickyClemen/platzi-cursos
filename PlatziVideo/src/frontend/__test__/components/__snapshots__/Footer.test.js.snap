// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> footer snapshot 1`] = `
<footer
  className="footer"
>
  <a
    href="/"
  >
    Términos de uso
  </a>
  <a
    href="/"
  >
    Declaración de privacidad
  </a>
  <a
    href="/"
  >
    Centro de ayuda
  </a>
</footer>
`;
